src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Navigation } from '../Navigation';
+import { Filters } from '../../types/Filters';
+import { client } from '../../utils/fetchClient';
+
+type Props = {
+  todos: Todo[]
+  setTodos: (callback: (todos: Todo[]) => Todo[]) => void
+  filterTodos: Filters
+  setFilterTodos: (option: Filters) => void
+  setErrorMessage: (message: string) => void
+  completedTodoCount: boolean
+  setLoadingItems: (id: (prevState: number[]) => number[]) => void
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  setTodos,
+  filterTodos,
+  setFilterTodos,
+  setErrorMessage,
+  completedTodoCount,
+  setLoadingItems,
+}) => {
+  const itemsLeft = (): number => {
```

Comment:
```suggestion
  const itemsLeft = () => {
```
I think that the type can be infered from return